src/templates.js

```diff
@@ -8,47 +11,89 @@ const p2pkh = parseAsm('OP_DUP OP_HASH160 <20-byte-pubkey-hash> OP_EQUALVERIFY O
 const p2sh = parseAsm('OP_HASH160 <20-byte-script-hash> OP_EQUAL', parseOpts)
 const p2wpkh = parseAsm('0 <20-byte-pubkey-hash>', parseOpts)
 const p2wsh = parseAsm('0 <32-byte-hash>', parseOpts)
+const p2ms = parseAsm('<OP_1-OP_16> <65-byte-pubkey> ... <OP_1-OP_16> OP_CHECKMULTISIG', parseOpts)
```

Comment:
Weren't were going to do this format? `<OP_M> <pubkey> ... <OP_N>`

Reply:
Probably should have the `OP_CHECKMULTISIG` also

Reply:
Isn't that what is there?  It differs in the labels since the labels are parsed for added constraints.  For example the `OP_1-OP_16` is read by the templating system to ensure the opcode is within that range.  The `65-byte-pubkey` label enforces that the pubkey is in fact 65 bytes to be exactly 65 bytes.